wa.user.styl

```diff
@@ -71,7 +71,7 @@
 @var color cyan    'Color: Cyan                 ' #5e8d87
 @var color green   'Color: Green                ' #7b823a
 @var color white   'Color: White                ' #ffffff
-@var color shadow  'Color: Shadow               ' #00000010
+@var color shadow  'Color: Shadow               ' #00000030
```

Comment:
Shadow still does not have the same spreading in many places:

1. Emojis:
![image](https://user-images.githubusercontent.com/20738487/70849311-863f4b00-1e85-11ea-81f2-637639b52dd3.png) 
![image](https://user-images.githubusercontent.com/20738487/70849322-948d6700-1e85-11ea-8dc0-74d537d5fb26.png)


2. Menus (ex: Main and Contact):
![image](https://user-images.githubusercontent.com/20738487/70849379-25fcd900-1e86-11ea-8890-d7709b3f25fd.png)
![image](https://user-images.githubusercontent.com/20738487/70849384-34e38b80-1e86-11ea-813b-8553876d5162.png)
![image](https://user-images.githubusercontent.com/20738487/70849362-ea620f00-1e85-11ea-915b-4c3bec347319.png)
![image](https://user-images.githubusercontent.com/20738487/70849368-f64dd100-1e85-11ea-9756-d20e6904f4ba.png)

Reply:
Most of your issues come from using two background colors instead of the default three, but I digress - that's your choice.

We've discussed this on Discord, and I've implemented something and been running/testing it locally for the last little while. Still looking for best way to use it with my other userstyles.
![image](https://user-images.githubusercontent.com/18245694/70852188-f3aba580-1e9e-11ea-9823-d7cf5d974347.png)

Reply:
Actually using 3 different yet similar colors:
**Color: Primary background** #272c35
**Color: Highlight background** #1f232a
**Color: Secondary background** #2e333e

The new view seems AMAZING!